src/main/java/mate/academy/bookstore/excepion/CustomGlobalExceptionHandler.java

```diff
@@ -42,4 +46,12 @@ private String getErrorMessage(ObjectError e) {
         }
         return e.getDefaultMessage();
     }
+
+    @ExceptionHandler(RegistrationException.class)
+    public ResponseEntity<Response<String>> handlerException(RegistrationException e) {
+        Response<String> response = new Response<>(e.getMessage());
+        response.setStatus(REGISTRATION_FAILED);
+        response.setBody(e.getMessage());
+        return new ResponseEntity<>(response,HttpStatus.BAD_REQUEST);
+    }
```

Comment:
```suggestion
    public ResponseEntity<Response<String>> handlerException(RegistrationException e) {
         return new ResponseEntity<>(e.getMessage(), HttpStatus.BAD_REQUEST);
    }
```
try if this works for you